Authentication/SignUp/View/SignupControllerDelegate.swift

```diff
@@ -41,7 +41,7 @@ extension SignupControllerDelegate {
     public var shouldDisplaySignupErrorWithAlert: Bool { return true }
     
     public func signupControllerWillExecuteSignUp(controller: SignupController) {
-        if let errorLabel = controller.signupView.signupErrorLabel {
+        if let errorLabel = controller.signupView.signUpErrorLabel {
             errorLabel.text = " "
```

Comment:
I think @FranDepascuali meant why are we changing from `signup` to `signUp`. I think it should be `signup` because it should be used as a noun.